Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.Data.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Represents the control that redistributes space between columns or rows of a Grid control.
+    /// </summary>
+    public partial class GridSplitter
+    {
+        // GridSplitter has special Behavior when columns are fixed
+        // If the left column is fixed, splitter will only resize that column
+        // Else if the right column is fixed, splitter will only resize the right column
+        private enum SplitBehavior
+        {
+            Split, // Both columns/rows are star lengths
+            Resize1, // resize 1 only
+            Resize2, // resize 2 only
+        }
+
+        // Only store resize data if we are resizing
+        private class ResizeData
```

Comment:
@dotMorten that's why I mentioned that I used some WPF splitter functions many times to check if it is possible to use or not, actually if it is violation of license agreement I was already written most of the logic for resizing 2 fixed columns and rows maybe I could revert to this point and continue working on my own code.
 
Thanks.